libsawtooth/src/store/receipt_store/mod.rs

```diff
@@ -41,11 +41,11 @@ impl FromBytes for TransactionReceipt {
 /// `TransactionReceiptStore` is a wrapper around an `OrderedStore` that stores
 /// `TransactionReceipt`s keyed by transaction IDs (`String`s) and indexed by `u64`s to provide
 /// ordering of transactions.
-pub struct TransactionReceiptStore(Box<dyn OrderedStore<String, TransactionReceipt, u64> + Send>);
+pub struct TransactionReceiptStore(Box<dyn OrderedStore<String, TransactionReceipt, u64>>);
```

Comment:
Typo in commit message `TransactionReceitStore` -> ` TransactionReceiptStore`

Reply:
Fixed